locale/hu.json

```diff
@@ -31,256 +31,256 @@
+            "announcement_title": "Szerver felhívás %{author}:",
+            "general_no_perms": "Ehhez nincs engedélyed!",
+            "dialog_empty_input": "Nem lehet üresen hagyni."
         },
         "frozen": {
-            "froze_player": "You have frozen the player!",
-            "unfroze_player": "You have unfrozen the player!",
-            "was_frozen": "You have been frozen by a server admin!"
+            "froze_player": "Lefagyasztottad a játékost!",
+            "unfroze_player": "Feloldottad a fagyasztást a játékoson!",
+            "was_frozen": "Lefagyasztott egy szerveradmin!"
         },
         "common": {
-            "cancel": "Cancel",
-            "submit": "Submit",
-            "error": "An error occurred",
-            "copied": "Copied to clipboard."
+            "cancel": "Vissza",
+            "submit": "Küldés",
+            "error": "Hiba lépett fel",
+            "copied": "Vágólapra másolva."
         },
         "page_main": {
             "tooltips": {
-                "tooltip_1": "Use %{key} to switch pages & the arrow keys to navigate menu items",
-                "tooltip_2": "Certain menu items have sub options which can be selected using the left & right arrow keys"
+                "tooltip_1": "%{key} a lapozáshoz és [NYILAK] a menüben való navigáláshoz",
+                "tooltip_2": "Néhány menü rendelkezik al opcióval  mely a bal és jobb nyíllal választható"
             },
             "player_mode": {
-                "title": "Player Mode",
+                "title": "Játékos mód",
                 "noclip": {
                     "title": "NoClip",
-                    "label": "Fly around",
-                    "success": "NoClip enabled"
+                    "label": "Körbe repülés",
+                    "success": "NoClip engedélyezve"
                 },
                 "godmode": {
                     "title": "God",
-                    "label": "Invincible",
-                    "success": "God Mode enabled"
+                    "label": "Halhatatlanság",
+                    "success": "God mód engedélyezve"
                 },
                 "normal": {
-                    "title": "Normal",
-                    "label": "Default mode",
-                    "success": "Returned to default player mode."
+                    "title": "Normál",
+                    "label": "Normál mód",
+                    "success": "Visszaállítva a normál játékos módba."
                 }
             },
             "teleport": {
-                "title": "Teleport",
-                "generic_success": "Sent you into the wormhole!",
+                "title": "Teleportálás",
+                "generic_success": "Átléptél egy féreglyukon!",
                 "waypoint": {
-                    "title": "Waypoint",
-                    "label": "Go to waypoint set",
-                    "error": "You have no waypoint set."
+                    "title": "Célpont",
```

Comment:
In all these, `célpont` could leave some people confused, I would recommend `úticél` instead.